packages/cli/src/commands/hydrogen/deploy.test.ts

```diff
@@ -176,18 +176,20 @@ describe('deploy', () => {
     expect(vi.mocked(renderSuccess)).toHaveBeenCalled;
   });
 
-  it('returns when there are uncommited changes', async () => {
+  it('errors when there are uncommited changes', async () => {
     vi.mocked(ensureIsClean).mockRejectedValue(
       new GitDirectoryNotCleanError('Uncommitted changes'),
     );
 
-    await oxygenDeploy(deployParams);
+    try {
+      await oxygenDeploy(deployParams);
+    } catch (error) {
+      expect(error).toBeInstanceOf(AbortError);
+      const abortError = error as AbortError;
+      expect(abortError.message).toBe('Uncommitted changes detected.');
+    }
```

Comment:
JFYI I think a simpler to `expect` errors in Vitest is:

```js
await expect(oxygenDeploy(deployParams)).rejects.toThrowError('Uncommited changes detected'));
```

Or this should also work:

```js
await expect(() => oxygenDeploy(deployParams)).toThrowError('Uncommited changes detected'));
```

Reply:
nice!